src/dictionaries/it.json

```diff
@@ -171,20 +171,20 @@
       "restorecodecopied": "Codice di ripristino copiato",
       "restoredaccess": "Accesso ripristinato",
       "whatisl3": "Cos’è un’identità di livello 3?",
-      "login-1": "Se hai messo in sicurezza la tua identità digitale puoi ripristinare l’accesso a IO e entrare nuovamente in app con tutte le tue identità.",
+      "login": "Se hai messo in sicurezza la tua identità digitale puoi ripristinare l’accesso a IO e entrare nuovamente in app con tutte le tue identità.",
```

Comment:
sorry but i have change this typo, can you fix into the code the login-1 with login? 

PS: i found the typo to fix following this path /io-web-profile/src/app/[locale]/(pages)/profileRestoreL3/page.tsx